accounting/decimal.go

```diff
@@ -1,36 +1,40 @@
 package accounting
 
 import (
-	"github.com/nspcc-dev/neofs-api-go/v2/accounting"
+	neofsproto "github.com/nspcc-dev/neofs-sdk-go/internal/proto"
+	protoaccounting "github.com/nspcc-dev/neofs-sdk-go/proto/accounting"
 )
 
 // Decimal represents decimal number for accounting operations.
 //
-// Decimal is mutually compatible with github.com/nspcc-dev/neofs-api-go/v2/accounting.Decimal
-// message. See ReadFromV2 / WriteToV2 methods.
+// Decimal is mutually compatible with [protoaccounting.Decimal] message. See
+// [Decimal.FromProtoMessage] / [Decimal.FromProtoMessage] methods.
 //
 // Instances can be created using built-in var declaration.
-//
-// Note that direct typecast is not safe and may result in loss of compatibility:
-//
-//	_ = Decimal(accounting.Decimal{}) // not recommended
-type Decimal accounting.Decimal
+type Decimal struct {
```

Comment:
Why can't it be an alias to `protoaccounting.Decimal`? On one hand, we remove a lot of overhead here. But on the other we still add some more. What this intermediate structure gives us?

Reply:
It'd make some sense if types/functions were different, maybe for some compatibility reasons. But this way I just don't see what problem it solves. Same thing, just some setters/getters. Some other conversions to `struct` are similar. In some cases we already have `struct` and it's somehow different, not a big problem there.

Reply:
it existed, I kept. We can purge this type. I dont mind

Reply:
for now, i can neither leave it wrapper nor drop and keep backcompat